x/executionlayer/client/rest/msgCreator.go

```diff
@@ -13,14 +14,12 @@ import (
 )
 
 type transferReq struct {
-	ChainID                    string `json:"chain_id"`
-	TokenContractAddress       string `json:"token_contract_address"`
-	SenderAddressOrNickname    string `json:"sender_address_or_nickname"`
-	RecipientAddressOrNickname string `json:"recipient_address_or_nickname"`
-	Amount                     uint64 `json:"amount"`
-	Fee                        uint64 `json:"fee"`
-	GasPrice                   uint64 `json:"gas_price"`
-	Memo                       string `json:"memo"`
+	BaseReq                    rest.BaseReq `json:"base_req"`
+	TokenContractAddress       string       `json:"token_contract_address"`
+	SenderAddressOrNickname    string       `json:"sender_address_or_nickname"`
```

Comment:
What do you think about the from address also go into baseReq?
I think from is an essential value when creating tx.

Reply:
https://github.com/hdac-io/friday/pull/70/commits/619b07f13770c808b45f2a8c12fb0efa8eb2fcff

Reply:
How about add to token_address & fee in baseReq?